app/Category.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    protected $table = 'categories' ;
+    public $timestamps = 'true';
+
+    public function musics(){
+        return $this->belongsToMany('App\Music', 'music_category', 'category_id', 'music_id');
```

Comment:
belongsToMany là quan hệ gì em?
Tại sao Category và Music lại có qun hệ này?

Reply:
belongsToMany là quan hệ nhiều nhiều ạ
em xem ở https://laravel.com/docs/5.8/eloquent-relationships#many-to-many
1 danh mục có nhiều bài hát, 1 bài hát cũng có thể thuộc nhiều danh mục ạ